### PR TITLE
Update collaboration-cycle-request.yml

### PR DESCRIPTION
**Pull request by sterkenburgsara**
Linked [Product operations terms and definitions](https://depo-platform-documentation.scrollhelp.site/getting-started/product-operations-terms-and-definition) page as description (beneath product name and feature name text boxes) to the collaboration cycle kickoff YAML ticket. Also updated language throughout ticket as needed.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/collaboration-cycle-request.yml [hunk 1]
```diff
@@ -68,15 +70,15 @@ body:
     id: start-date
     attributes:
       label: When did/will you start working on this product/feature?
-    validations:
+        validations:
```
Comment (briandeconinck) [inline .github/ISSUE_TEMPLATE/collaboration-cycle-request.yml:73]: Looks like an extra indentation got added here. GitHub's form engine will choke on that.

**Comment by briandeconinck** [comment]
@sterkenburgsara Besides the item I flagged about the indentation, I think this looks good. Do you want to try copying it to a personal repo to preview/share with the group before merging? If so, I can help you set it up in a personal repo -- just let me know.

File: .github/ISSUE_TEMPLATE/collaboration-cycle-request.yml (hunk 1 above)
Comment (sterkenburgsara) [reply .github/ISSUE_TEMPLATE/collaboration-cycle-request.yml:73]: Thanks for catching/flagging!

**Comment by sterkenburgsara** [comment]
@briandeconinck yes, let's connect. I'll get something on the calendar. Thank you!

**Review by bethdeconinck — APPROVED** [review]
This all looks good to me!

**Pull request merged**